lighthouse-core/gather/driver.js

```diff
@@ -809,11 +785,9 @@ class Driver {
    * @return {Promise<Array<Element>>} The found elements, or [], resolved in a promise
    */
   async querySelectorAll(selector) {
-    /** @type {LH.Crdp.DOM.GetDocumentResponse} */
-    const documentResponse = await this.sendCommand('DOM.getDocument');
+    const documentResponse = await this.sendCommand('DOM.getDocument', {});
```

Comment:
I thought the `| void` would let us omit the parameters if they're all optional?

Reply:
whoops, yes, those were from an earlier version of the PR